.gitignore

```diff
@@ -133,9 +109,26 @@ cmake_install.cmake
 /addons/adsp.*
 /addons/peripheral.*
 /addons/game.*
+/addons/kodi.binary.global.audioengine/addon.xml
+/addons/kodi.binary.global.filesystem/addon.xml
+/addons/kodi.binary.global.general/addon.xml
+/addons/kodi.binary.global.gui/addon.xml
+/addons/kodi.binary.global.main/addon.xml
+/addons/kodi.binary.global.network/addon.xml
+/addons/kodi.binary.instance.adsp/addon.xml
+/addons/kodi.binary.instance.audiodecoder/addon.xml
+/addons/kodi.binary.instance.audioencoder/addon.xml
+/addons/kodi.binary.instance.game/addon.xml
+/addons/kodi.binary.instance.imagedecoder/addon.xml
+/addons/kodi.binary.instance.inputstream/addon.xml
+/addons/kodi.binary.instance.peripheral/addon.xml
+/addons/kodi.binary.instance.pvr/addon.xml
+/addons/kodi.binary.instance.screensaver/addon.xml
+/addons/kodi.binary.instance.vfs/addon.xml
+/addons/kodi.binary.instance.videocodec/addon.xml
+/addons/kodi.binary.instance.visualization/addon.xml
```

Comment:
does it make sense to use a wildcard here?
i.e. /addons/kodi.binary.instance.*/addon.xml